packages/sdk-node/src/clerkExpressRequireAuth.ts

```diff
@@ -29,22 +24,11 @@ export const createClerkExpressRequireAuth = (createOpts: CreateClerkExpressMidd
         options,
       });
       decorateResponseWithObservabilityHeaders(res, requestState);
-      if (requestState.isUnknown) {
-        return handleUnknownCase(res, requestState);
-      }
-      if (requestState.isInterstitial) {
-        const interstitial = await loadInterstitial({
-          clerkClient,
-          requestState,
-        });
-        if (interstitial.errors) {
-          // Temporarily return Unauthenticated instead of the interstitial errors since we don't
-          // want to expose any internal error (possible errors are http 401, 500 response from BAPI)
-          // It will be dropped with the removal of fetching remotePrivateInterstitial
-          next(new Error('Unauthenticated'));
-          return;
-        }
-        return handleInterstitialCase(res, requestState, interstitial.data);
+
+      if (requestState.status === AuthStatus.Handshake) {
+        // TODO: Handle handshake
```

Comment:
@nikosdouvlis Reminder for this ☝️

Reply:
@SokratisVidros 
Posting here for posterity - this will be handled with a 2nd PR once we've completed testing the nextjs integration.